### PR TITLE
Update UG for assignment

### PR DESCRIPTION
**Pull request by waihongteh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -521,8 +535,9 @@ Format: `unmarkAsg INDEX n/ASSIGNMENT_TITLE`
 Command Details & Constraints:
 * This command is the opposite of mark command.
 * Using the given index of the student, TrackMate will mark his/her assignment status as not completed.
-* Assignment with the given title must exist.
+* Assignment with the given non-empty title must exist.
```
Comment (tansanchian) [inline docs/UserGuide.md:538]: Good to see that you spot this small thing that may lead to confusion!

**Review by tansanchian — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -759,6 +759,40 @@ testers are expected to do more *exploratory* testing.
     1. Test case: `markAtt s/A1234567X t/T1001 d/21-10-2023`<br>
        Expected: Error message indicating that the date format is invalid. Correct format should be "YYYY-MM-DD".
 
+### Marking/Unmarking assignment
```
Comment (Fui03) [inline docs/DeveloperGuide.md:762]: Good to see that you give examples in developer guide!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -188,7 +188,7 @@ First-time users are strongly encouraged to read the Getting Started section bef
    Some example commands you can try:
 
 * `list` : Lists all contacts.
-* `addTut c/CS2103T c/T1001` : Adds a tutorial of name `CS2103T` and tutorial id `T1001`.
+* `addTut tn/CS2103T c/T1001` : Adds a tutorial of name `CS2103T` and tutorial id `T1001`.
```
Comment (Fui03) [inline docs/UserGuide.md:191]: Good to see that the prefix for tutorial name is corrected!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -271,7 +271,7 @@ Essentially they are to be supplied by the user.
...
+| `ASSIGNMENT_TITLE`    | `n/`             | Specifies the name of an assignment. <br/><br/> **Requirements:** <ul><li>Names may contain any characters, including alphanumeric characters, whitespaces and special symbols.</li><li>Each `ASSIGNMENT_TITLE` must be unique.</li><li>`ASSIGNMENT_TITLE` is case sensitive.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (Fui03) [inline docs/UserGuide.md:274]: Is it possible to include all kind of special symbols including '/' ?

**Review by Fui03 — COMMENTED** [review]
Please make sure that the assignment title's constraints are tested and updated accordingly.

**Pull request merged**